api/users/education-schema.json

```diff
@@ -0,0 +1,52 @@
+{
+	"type": "array",
```

Comment:
Super minor, but could you configure your editor to use spaces instead of tabs for json files? This file seems a bit overindented.